chore: update docs about cookie security (#96)
fastify/fastify-cookie: README.md

+ `options`: an options object as described in the [cookie serialize][cs] documentation
  with a extra param "signed" for signed cookie

#### Securing the cookie

> Following are _some_ of the precautions that should be taken to ensure the integrity of an application:

Reviewer: I would also add to use HTTPS.

> Following are _some_ of the precautions you could take to ensure safety of your application.
> 
> - It's important to use `options.httpOnly` cookies to prevent attacks like XSS.
> - Use signed cookies (`options.signed`) to ensure they are not getting tampered with on client-side by an attacker.
> - Use `__Host-` [Cookie Prefix](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#Attributes) to avoid Cookie Tossing attacks.

Reviewer 2: ```suggestion
Following are _some_ of the precautions that should be taken to ensure the integrity of an application:

- It's important to use `options.httpOnly` cookies to prevent attacks like XSS.
- Use signed cookies (`options.signed`) to ensure they are not getting tampered with on client-side by an attacker.
- Use `__Host-` [Cookie Prefix](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#Attributes) to avoid Cookie Tossing attacks.
```
